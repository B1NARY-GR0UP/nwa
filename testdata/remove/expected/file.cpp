#include <iostream>
#include <string>

class Greeter {
private:
    std::string message;
public:
    Greeter(const std::string& msg) : message(msg) {}
    void greet() {
        std::cout << message << std::endl;
    }
};

int main() {
    Greeter greeter("Hello, World!");
    greeter.greet();
    return 0;
}