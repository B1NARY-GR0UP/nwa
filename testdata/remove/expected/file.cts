function sayHello(name: string): string {
    return `Hello, ${name}!`;
}

const greeting: string = "World";

module.exports = {
    sayHello,
    greeting
};