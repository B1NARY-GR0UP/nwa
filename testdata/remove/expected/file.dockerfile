FROM ubuntu:22.04

# Metadata
LABEL maintainer="example@example.com"
LABEL version="1.0"

# Install requirements
RUN apt-get update && apt-get install -y \
    curl \
    && rm -rf /var/lib/apt/lists/*

# Set working directory
WORKDIR /hello

# Create hello world script
RUN echo '#!/bin/sh\necho "Hello, World!"' > /hello/hello.sh && \
    chmod +x /hello/hello.sh

# Set entry point
ENTRYPOINT ["/hello/hello.sh"]