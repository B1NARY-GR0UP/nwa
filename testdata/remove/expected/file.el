(defun hello-world ()
  "Print a hello world message."
  (interactive)
  (message "Hello, World!"))

;; Define a variable
(defvar greeting "Hello, Emacs Lisp!"
  "A simple greeting message.")

;; Call the function
(hello-world)