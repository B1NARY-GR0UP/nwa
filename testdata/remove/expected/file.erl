-module(hello).
-export([hello_world/0]).

hello_world() ->
    io:format("Hello, World!~n").

% A simple function with pattern matching
greet(Name) ->
    io:format("Hello, ~s!~n", [Name]).