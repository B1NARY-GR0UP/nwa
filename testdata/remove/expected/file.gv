digraph HelloWorld {
    node [shape=box];
    Hello -> World [label="says"];
    World [style=filled, color=lightblue];
    label = "Hello World Graph";
}