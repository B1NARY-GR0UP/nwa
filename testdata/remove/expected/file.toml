title = "Hello World Example"

[greeting]
text = "Hello, World!"
language = "English"

[author]
name = "Example User"
role = "Developer"

[dependencies]
lib1 = "1.0.0"
lib2 = "2.3.4"